recipes/cget/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "cget" %}
+{% set version = "0.1.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 2a7913b601bec615208585eda7e69998a43cc17080d36c2ff2ce8742c9794bf6
+
+build:
+  number: 0
+  entry_points:
+    - cget = cget.cli:cli
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - click >=6.6
+    - six >=1.10
+    - subprocess32  # [py27]
+  run:
+    - python
```

Comment:
```suggestion
    - python >=3
```

Reply:
@chrisburr good to keep this as `python` only without version reference (e.g. `>=2.7`)?

Reply:
Either is fine, including the reference is nice but in practice the entirety of conda-forge implicitly has this version constraint.